rhcephcompose/compose.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, conf):
         self.product_version = conf['product_version']
         # Extra files to put at the root of the compose
         self.extra_files = conf['extra_files']
+        # Whether sources composition should be skipped
+        self.no_sources = conf['no_sources']
```

Comment:
Yeah, that's fine. Although I don't think two is too many. :stuck_out_tongue_winking_eye: